www/js/survey/enketo/enketoHelper.ts

```diff
@@ -69,15 +245,14 @@ export function saveResponse(surveyName: string, enketoForm: Form, appConfig, op
         jsonDocResponse,
       };
       if (opts.timelineEntry) {
-        let timestamps = EnketoSurveyAnswer.resolveTimestamps(xmlDoc, opts.timelineEntry);
+        let timestamps = resolveTimestamps(xmlDoc, opts.timelineEntry);
         if (timestamps === undefined) {
           // timestamps were resolved, but they are invalid
           return new Error(i18next.t('survey.enketo-timestamps-invalid')); //"Timestamps are invalid. Please ensure that the start time is before the end time.");
         }
         // if timestamps were not resolved from the survey, we will use the trip or place timestamps
-        timestamps ||= opts.timelineEntry;
-        data.start_ts = timestamps.start_ts || timestamps.enter_ts;
-        data.end_ts = timestamps.end_ts || timestamps.exit_ts;
+        data.start_ts = timestamps.start_ts || opts.timelineEntry.enter_ts;
+        data.end_ts = timestamps.end_ts || opts.timelineEntry.exit_ts;
```

Comment:
I was curious about this change and briefly investigated to make sure it wouldn't cause issues.
I found that it was done while resolving merge conflicts when the Prettier reformat was merged in. `timestamps` never gets reassigned, but it is not used after this, so the behavior of the code will not change. It's fine like this (and cleaner anyway)